web/client/translations/data.en-US.json

```diff
@@ -2243,6 +2244,7 @@
             "imageSrcLoadError": "The provided image source could not be loaded. Try with a different url or ensure the source is reachable by MapStore",
             "imageSrcInvalidBase64": "This is not a valid base64 image. Some style renderer does not support this type of source, you could use an external url source instead",
             "imageSrcNotSupportedBase64Image": "Some style renderer does not support base64 images even if the current source is valid",
+            "imageFormatEmpty": "Parser unable to identify format of the image even when the source is valid. Please select the format of the image provided from the dropdown",
```

Comment:
I propose to change this message with something less technical:

`The image format cannot be identified. Please select the image format from the available ones in the 'Format' dropdown.`